spec/requests/my_request_request_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "MyRequests", type: :request do
```

Comment:
Remove this if you are not going to use it

Reply:
done